src/rkr/tracing/Tracer.cc

```diff
@@ -552,15 +553,24 @@ shared_ptr<Process> Tracer::launchTraced(Build& build, const shared_ptr<Command>
         bpf.push_back(BPF_STMT(BPF_RET | BPF_K, SECCOMP_RET_ALLOW));
         bpf.push_back(BPF_STMT(BPF_RET | BPF_K, SECCOMP_RET_TRACE));
 
-        //ERIC WAS HERE
-      } else if (SyscallTable<Build>::get(i).isBlocked()) {
-          bpf.push_back(BPF_JUMP(BPF_JMP | BPF_JEQ | BPF_K, i, 0, 1));
-          //bpf.push_back(BPF_STMT(BPF_RET | BPF_K, SECCOMP_RET_ALLOW));
-          //bpf.push_back(BPF_STMT(BPF_RET | BPF_K, SECCOMP_RET_ERRNO | (SIGSYS & SECCOMP_RET_DATA)));
-          bpf.push_back(BPF_STMT(BPF_RET | BPF_K, SECCOMP_RET_KILL_PROCESS));
-          //bpf.push_back(BPF_STMT(BPF_RET | BPF_K, SECCOMP_RET_TRACE));
       } else {
+        //ERIC
         if (SyscallTable<Build>::get(i).isTraced()) {
+          std::vector<std::string> blockedCalls{"socket", "socketpair", "setsockopt", "getsockopt", "getsockname", "getpeername", "bind", "listen", "accept", "accept4", "connect", "shutdown", "recvfrom", "recvmsg", "recvmmsg", "sendto", "sendmsg", "sendmmsg", "sethostname", "setdomainname", "bpf"};
```

Comment:
Could this be added in some header file?

Reply:
Yeah, I will move them to Tracer.hh as a private variable.